zc_common/remote_resource/tests.py

```diff
@@ -12,11 +12,54 @@ class ResponseTestCase(APITestCase):
     FAILURE_HIGH_LEVEL_KEYS = ['errors']
     FAILURE_DATA_KEYS = ['status', 'source', 'detail']
 
+    INSTANCE_TOP_LEVEL_KEYS = ["type", "id", "attributes"]
```

Comment:
While Python has no recommendation about whether you should use single or double quotes for strings, they do suggest that you try to be consistent.

My personal preference is single quotes for most everything aside from strings that will be seen by a user, which I will use double quotes for as well as block comments/docstrings. I will also switch depending on whether or not the string has a single or double quote inside of it (in order to avoid having to specifically escape quote marks).

I've gone with single quotes in things I've committed to this package to match what we're doing in timezone.py and to match my personal preference, so I'd probably recommend changing these to single quotes for consistency. There's not anything technically wrong with having them as double quotes though.